src/qt_gui/cheats_patches.cpp

```diff
@@ -39,7 +39,7 @@ CheatsPatches::CheatsPatches(const QString& gameName, const QString& gameSerial,
       m_gameSize(gameSize), m_gameImage(gameImage), manager(new QNetworkAccessManager(this)) {
     setupUI();
     resize(500, 400);
-    setWindowTitle(tr("Cheats / Patches"));
+    setWindowTitle(tr("Cheats / Patches for ") + m_gameName);
```

Comment:
This should have been `setWindowTitle(tr("Cheats / Patches for %1").arg(m_gameName));` for better translations.